### DISCUSSION TITLE
Why isn't my JSON showing as formatted?

### DISCUSSION THREAD

**Discussion by RezaSeedin** [burkybang/SuiteAdvanced-Feedback#59] [Q&A] [answered]
Hi @burkybang,

The JSON preview is not working. Not sure when it broke.

![image](https://user-images.githubusercontent.com/18523110/197085748-21fc65bd-9a6b-4053-8584-9a417be53657.png)

**Comment by burkybang** [comment, marked as answer]
Don't format the JSON yourself. The HTML line breaks interfere causing it not to be valid JSON.

**Reply by RezaSeedin** [reply]
Ohhh damm. thanks man

**Reply by burkybang** [reply]
No prob. I'll look into possibly getting it to ignore the line breaks at some point.

**Reply by burkybang** [reply]
I got it working. It'll be in the next version!

**Reply by burkybang** [reply]
![image](https://user-images.githubusercontent.com/927886/197092229-c689232b-ba4a-4be1-9a50-4a96fff32e81.png)
